app/Http/Controllers/RestaurantController.php

```diff
@@ -20,10 +21,47 @@ class RestaurantController extends Controller
 
     private $restaurant;
     private $review;
+    private $restaurantphoto;
+    private $foodtype;
+    private $areatype;
+    private $feature;
+    private $featuretype;
 
-    public function __construct(Restaurant $restaurant, Review $review){
+    public function __construct(Restaurant $restaurant, Review $review, RestaurantPhoto $restaurantphoto, FoodType $foodtype, AreaType $areatype, Feature $feature, FeatureType $featuretype){
         $this -> restaurant = $restaurant;
         $this -> review = $review;
+        $this -> restaurantphoto = $restaurantphoto;
```

Comment:
Please prevent having a space before and after ->

$this -> restaurantphoto change to $this->restaurantphoto